ruspy/simulation/simulation.py

```diff
@@ -51,51 +52,25 @@ def simulate(init_dict, ev_known=None, shock=None):
     beta = init_dict["beta"]
     num_periods = init_dict["periods"]
     params = np.array(init_dict["params"])
-    if "real_trans" in init_dict.keys():
-        real_trans = np.array(init_dict["real_trans"])
-    else:
-        real_trans = np.array(init_dict["known_trans"])
     if init_dict["maint_func"] == "linear":
```

Comment:
?!

Reply:
As I now require the transition matrix, there is no need for specifying any probabilities in the init dict.

Reply:
As you probably ment the maintenance function, I excuse the misunderstanding :)